scanapp/models.py

```diff
@@ -107,7 +121,9 @@ def __str__(self):
 
     scan_result = models.ForeignKey(ScanResult)
     file_path = models.CharField(max_length=400)
-
+    
+    class Meta:
+        verbose_name_plural = 'Scan File Info'
```

Comment:
This meta is not needed IMHO. Also why do you suffix all your models with `Info`?if these are objects they represent something not SomethingInfo